Insterm/SSHTerminalView.swift

```diff
@@ -38,6 +39,28 @@ class SSHTerminalView: TerminalView, TerminalViewDelegate, NMSSHChannelDelegate
         print("didReadError: \(error)")
     }
     
+    func isConnected() -> Bool {
+        if (connection != nil) {
+            return connection!.isConnected()
+        } else {
+            return false
+        }
+    }
+    
+    // ピンチイン・アウトでフォントサイズを変更
+    @objc func pinchInOut(_ gestureRecognizer: UIPinchGestureRecognizer) {
+        if gestureRecognizer.state == .began || gestureRecognizer.state == .changed {
+            let newSize = font.pointSize * gestureRecognizer.scale
+            gestureRecognizer.scale = 1.0
+            
+            if newSize < 10 || newSize > 40 {
```

Comment:
好みに近いのですが、Swiftでは条件チェックはguardで書くことが多いです。

```suggestion
            guard newSize < 10 || newSize > 40  else { return } 
```

Reply:
単純にguardに書き換えた場合に動かなかったため、下記のように書き換えてみました。
いかがでしょうか。
```
guard newSize < 10 || newSize > 40 else {
 font = UIFont.monospacedSystemFont(ofSize: newSize, weight: .regular)
 return
}
```
ちなみに下記のような書き方をした場合は動きませんでした。
```
guard newSize > 10 || newSize < 40 else { return }
font = UIFont.monospacedSystemFont(ofSize: newSize, weight: .regular)
```
想定ではどちらも同じ動きをすると思ってましたが、２番目の書き方ですと条件式の数値以上に拡大・縮小されました。

Reply:
guardに書き換える場合、元の条件式

```
if newSize < 10 || newSize > 40 { return } 
```

の否定になるので、

```
// !(newSize < 10 || newSize > 40)の対偶
guard newSize > 10 && newSize < 40 else { return } ※1
font = UIFont.monospacedSystemFont(ofSize: newSize, weight: .regular)
```

で同じ動作になるかと思いますが、いかがでしょう？

!(newSize < 10 || newSize > 40 )の対偶です